packages/react-ui/src/Stripe/index.js

```diff
@@ -3,10 +3,10 @@ import styled, { css } from "styled-components";
 
 import { box } from "../theme.js";
 
-export const Stripe = styled.div`
+export const Stripe = styled.span`
   position: absolute;
   background-color: ${({ variant, theme }) => theme[variant]};
-  border-radius: ${({ rounded }) => (rounded ? box.borderRadius : "none")};
+  border-radius: ${({ rounded }) => (rounded ? box.borderRadius : "0")};
```

Comment:
Je pense la props on peut la mettre en conditional : https://stackoverflow.com/questions/48502647/conditional-rendering-in-styled-components

```
const Stripe = styled.span`
  position: absolute;
  background-color: ${({ variant, theme }) => theme[variant]};
  ${({ rounded }) => rounded && `
    borderRadius: box.borderRadius;
  `}
`;
```